tst/EnergyPlus/unit/AirflowNetworkBalanceManager.unit.cc

```diff
@@ -8210,4 +8212,34 @@ TEST_F(EnergyPlusFixture, BasicAdvancedSingleSided)
     }
 }
 
+TEST_F(EnergyPlusFixture, AFN_CheckNumOfFansInAirLoopTest)
+{
+    // #6748
```

Comment:
This isn't needed, we can use our version control to find when this unit test was added and track down the issue it fixed.  Remove it.

Reply:
@Myoldmopar  Removed it.